Resources/Private/Partials/Form/ValidationErrors.html

```diff
@@ -0,0 +1,16 @@
+<f:form.validationResults>
+  <f:if condition="{validationResults.flattenedErrors}">
+    <div class="alert alert-danger">
+      <p>
+        <f:translate key="validationErrors.errors">There where some errors with your input:</f:translate>
```

Comment:
You forgot to add `validationErrors.errors` to the translation file once again. (And remove `resetPassword.errors` again.)